pom.xml

```diff
@@ -875,12 +875,16 @@
                 <plugin>
```

Comment:
I agree all commits should build when merged to master, so that bisects work. I introduced the changes in this order so that I see the static checks I introduced catch anything. I left it this way so far so that anyone can easily re-verify it. Will now rearrange the commits.